### PR TITLE
Duplicates in ng2-select, select by Enter in multiple mode

### PR DESCRIPTION
**Pull request by aslubsky**
Fix for https://github.com/valor-software/ng2-select/issues/72

### CODE REVIEW

**Comment by valorkin** [comment]
Looks good to me!
But please squash commits and force push to your branch
And rename commits in angular2 commit name convention
fix(multi select): duplicates, select by Enter in multiple mode

**Comment by aslubsky** [comment]
Ok, I've created https://github.com/valor-software/ng2-select/pull/82 from another account. Because my fork https://github.com/aslubsky/ng2-select already has more changes. I've added "tagging" functionality and it's in some dirty state. I'll create new PR after testing and refactoring

File: .gitignore [hunk 1]
```diff
@@ -13,11 +13,5 @@ npm-debug.log
 /demo/**/*.js
 /demo/**/*.js.map
 /demo/**/*.d.ts
-/components/**/*.js
```
Comment (valorkin) [inline .gitignore:16]: hard to follow PR, generated files should be excluded

**Comment by aslubsky** [comment]
Close as duplicate of https://github.com/valor-software/ng2-select/pull/82
